compiler/optimizer/CMakeLists.txt

```diff
@@ -103,4 +103,8 @@ compiler_library(optimizer
 	${CMAKE_CURRENT_LIST_DIR}/ValuePropagationCommon.cpp
 	${CMAKE_CURRENT_LIST_DIR}/TrivialDeadBlockRemover.cpp
 	${CMAKE_CURRENT_LIST_DIR}/FEInliner.cpp
+	${CMAKE_CURRENT_LIST_DIR}/abstractinterpreter/AbsValue.cpp
+	${CMAKE_CURRENT_LIST_DIR}/abstractinterpreter/AbsOpStack.cpp
+	${CMAKE_CURRENT_LIST_DIR}/abstractinterpreter/AbsLocalVarArray.cpp
+	${CMAKE_CURRENT_LIST_DIR}/abstractinterpreter/AbsState.cpp
```

Comment:
@dnakamura is this ok or should we create a new CMakeLists.txt inside the `abstractinterpreter` directory?

Reply:
I'm ok with either